mmv1/third_party/terraform/services/container/node_config.go.tmpl

```diff
@@ -756,6 +757,12 @@ func schemaNodeConfig() *schema.Schema {
 								ForceNew: true,
 								Description: `Whether the node should have nested virtualization enabled.`,
 							},
+							"performance_monitoring_unit": {
+								Type: schema.TypeString,
+								Optional: true,
+								ValidateFunc: verify.ValidateEnum([]string{"ARCHITECTURAL", "STANDARD", "ENHANCED"}),
```

Comment:
Is it possible new values will be added in the future? If they will, we may want to leave off validation here.